### ISSUE TITLE
Accessibility: MAS40B: Inspect: Name property is too verbose for the Extension links 

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#44]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open [https://code.visualstudio.com/docs ](https://code.visualstudio.com/docs )in EDGE.
2. Navigate to the extension links available under "Top Extensions" like C#, Python etc.
3. Observe the "Name" property in Inspect
**This issue is observed for all the extension links available in the page [refer screenshot]** 
**_Same issue is also observed with the links:_**
[https://code.visualstudio.com/docs/languages/overview](https://code.visualstudio.com/docs/languages/overview) [refer screenshot]
https://code.visualstudio.com/docs/runtimes/nodejs [refer screenshot]

**Actual Result:**
The "Name" property is too verbose.
**_Narrator Behaviour: (for C# extension link)_**
C#1103.3Kms-vscodeC# for Visual Studio Code powered by OmniSharp link

**Expected Result:**
Name property should indicate the purpose of the links. So, Name property should be meaningful and not verbose.
**_Eg: C# Extension Link_**

**User Impact:**
Visually impaired users  will not get to know the purpose of the link as the screen reader will not announce meaningful name when the user arrives at the extension links.
Providing meaningful name property enables the compatibility with the Screen Reader, which may be used by people with disabilites.

**Screenshots:**
![extension links](https://cloud.githubusercontent.com/assets/24825441/21718111/13d3693a-d43b-11e6-9a3d-50e4cbd778c8.png)
![name property is not meaningful for extension links](https://cloud.githubusercontent.com/assets/24825441/21718108/13d1dfa2-d43b-11e6-8e43-152419bd33ed.png)
![narrator buddy screenshot for c extension link](https://cloud.githubusercontent.com/assets/24825441/21718110/13d212a6-d43b-11e6-84c9-96e6ac26cba6.png)
![visualstudio com_docs_languages_screenshot](https://cloud.githubusercontent.com/assets/24825441/21718109/13d1e8da-d43b-11e6-9f7b-1ad80069cc30.png)
![visualstudio com_docs_runtimes](https://cloud.githubusercontent.com/assets/24825441/21718107/13d18926-d43b-11e6-992d-86444965246a.png)

**Comment by waderyan** [comment]
Fixed
